### PR TITLE
added authentication plus improved loading areas/groups/programs

### PR DESCRIPTION
**Pull request by grijzea**

### CODE REVIEW

File: src/components/LandingPage.vue [hunk 1]
```diff
@@ -36,9 +36,8 @@
                 src="../../public/arrow.png"
             ></v-img>
           </v-col>
-          <v-col md="2"></v-col>
-          <v-col md="1">
-            <v-btn to="SelectProgram" text color="primary" @click="e1 = 2">Select program(s)
+          <v-col md="3">
+            <v-btn to="SelectProgram" text color="primary" @click="e1 = 2">Sel edct program(s)
```
Comment (Carsopre) [inline src/components/LandingPage.vue:40]:
```suggestion
            <v-btn to="SelectProgram" text color="primary" @click="e1 = 2">Select program(s)
```

File: src/views/LoginPage.vue [hunk 2]
```diff
@@ -1,33 +1,50 @@
...
-        </v-row>
-      </v-card-text>
-    </v-card>
-  </div>
+  <v-row>
+    <v-col md="3">
+    </v-col>
+    <v-col md="6">
+      <v-card>
+        <v-card-title class="justify-center">Please login with your user name and password</v-card-title>
```
Comment (Carsopre) [inline src/views/LoginPage.vue:7]:
```suggestion
        <v-card-title class="justify-center">Please login with your username and password</v-card-title>
```

**Review by Carsopre — CHANGES_REQUESTED** [review]

File: src/views/LoginPage.vue (hunk 2 above)
Comment (grijzea) [reply src/views/LoginPage.vue]: fixed

File: src/components/LandingPage.vue (hunk 1 above)
Comment (grijzea) [reply src/components/LandingPage.vue]: also fixed

**Review by Carsopre — APPROVED** [review]

**Pull request merged**